install_config/router/customized_haproxy_router.adoc

```diff
@@ -115,21 +115,25 @@ The *define* action names the file that will contain the processed template.
 [cols="2*", options="header"]
 |===
 |Function | Meaning
-|`*endpointsForAlias(alias ServiceAliasConfig, svc ServiceUnit) []Endpoint*` | Returns the list of valid endpoints.
-|`*env(variable, default string) string*` | Tries to get the named environment variable from the pod. Returns the second
-argument if the variable cannot be read or is empty.
+|`*processEndpointsForAlias(alias ServiceAliasConfig, svc ServiceUnit, action string) []Endpoint*` |
+Returns the list of valid endpoints. When action is "shuffle", the order of endpoints is randomized.
+|`*env(variable string, default ...string) string*` | Tries to get the named environment variable from the pod.
+If it is not defined or empty, it returns the optional second argument.
```

Comment:
This doesn't describe the behaviour when no second argument is given, or when more than two arguments are given.

Reply:
Changed. Otherwise it returns the empty string. Other arguments are ignored.